source/ocsp-support/ocsp-support.rst

```diff
@@ -194,9 +194,21 @@ This boolean option determines whether a MongoClient should refrain from
 reaching out to an OCSP endpoint i.e.  whether non-stapled OCSP should
 be disabled.  When set to true, a driver MUST NOT reach out to OCSP
 endpoints. When set to false, a driver MUST reach out to OCSP
-endpoints if needed (as described in 
+endpoints if needed (as described in
 `Specification: Suggested OCSP Behavior <ocsp-support.rst#id1>`__).
-This option MUST default to false.
+This option MUST default to false except in the case that a driver' TLS
+library exhibits hard-fail behavior (i.e. the driver fails the
+`"Soft Fail Test
+"<https://github.com/mongodb/specifications/tree/master/source/ocsp-support/tests#integration-tests-permutations-to-be-tested>`__
+).
+
+In the case of a driver whose TLS library exhibits hard-fail behavior,
+this value MUST default to true and a driver MUST document this
+behavior.  If this hard-failure behavior is specific to a particular
+platform (e.g. the TLS library hard-fails only on Windows) then this
+boolean MUST default to true only on the platform where the exhibits
```

Comment:
where the -> where it

Reply:
Oops, thanks! I changed "where the" to be  "where the driver" since I felt "it" was very mildly ambiguous.